events/importer/kulke.py

```diff
@@ -979,6 +979,24 @@ def _handle_removed_events(
                 count,
             )
 
+        self._handle_referenced_deleted_events(elis_event_ids, begin_date)
```

Comment:
Is it possible that we will bring back events that in turn should prevent some super events from being deleted in the above super event soft delete thing? Ie. should this `_handle_referenced_deleted_events()` be run before line 961?

Reply:
Very nice catch, sir. I moved the restoring part above the super event deletion and added a test for this case.

Reply:
To be exact, I was talking about a situation where there is a super event which is erroneously deleted because its sub-events haven't been restored yet. And at least the new test doesn't address that particularly 🙂 

But as said, I'm not sure if that is even a possible scenario in practice 😄